src/repository/screens/repository-file.screen.js

```diff
@@ -105,17 +113,29 @@ class RepositoryFile extends Component {
                 </Text>
               </View>
 
-              {!this.isImage(fileType) &&
+              {!this.isImage(fileType) && !this.isKnownType(fileType) &&
```

Comment:
Yes, nice one. 
And I can also remove this useless `isKnownType` function, and replace it by the `getLanguage` one imported. What do you think about it?